src/components/player-info.js

```diff
@@ -120,12 +120,16 @@ AFRAME.registerComponent("player-info", {
     this.updateDisplayNameFromPresenceMeta(e.detail);
   },
   updateDisplayNameFromPresenceMeta(presenceMeta) {
+    this.permissions = presenceMeta.permissions;
     this.displayName = presenceMeta.profile.displayName;
     this.identityName = presenceMeta.profile.identityName;
     this.isRecording = !!(presenceMeta.streaming || presenceMeta.recording);
     this.isOwner = !!(presenceMeta.roles && presenceMeta.roles.owner);
     this.applyDisplayName();
   },
+  can(perm) {
```

Comment:
The `can` function in `hub-channel.js` has a nice `VALID_PERMISSIONS` check that we might want to use here as well, if we continue to use this function more widely.